widget/embedded/src/components/SwapDetailsAlerts/SwapDetailsAlerts.tsx

```diff
@@ -47,7 +51,7 @@ export function SwapDetailsAlerts(props: SwapAlertsProps) {
                     variant="ghost"
                     size="xsmall"
                     onClick={() => window.open(explorerUrl.url, '_blank')}>
-                    <LinkIcon size={16} />
+                    <LinkIcon size={12} />
                   </IconButton>
                 )
               }
```

Comment:
![Screenshot](https://github.com/rango-exchange/rango-client/assets/123297941/02bbac7f-4823-4209-b9a7-d659773ac4af)

I thin it should be handled in this condition or somewhere else `(waitingForApproval && !step.explorerUrl?.length)) `